### PR TITLE
feat(dws): add workload plan resource support

### PR DESCRIPTION
**Pull request by jinyangyang222**
**What this PR does / why we need it**:
add workload plan resource support
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
add workload plan resource support
```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST="./huaweicloud/services/acceptance/dws" TESTARGS="-run TestAccResourceWorkLoadPlan_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dws -v -run TestAccResourceWorkLoadPlan_basic -timeout 360m -parallel 4
=== RUN   TestAccResourceWorkLoadPlan_basic
=== PAUSE TestAccResourceWorkLoadPlan_basic
=== CONT  TestAccResourceWorkLoadPlan_basic
--- PASS: TestAccResourceWorkLoadPlan_basic (1316.41s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dws       1316.444s
```

### CODE REVIEW

File: huaweicloud/services/dws/resource_huaweicloud_dws_workload_plan.go [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+	if err != nil {
+		return []*schema.ResourceData{d}, fmt.Errorf("error creating DWS client: %s", err)
+	}
+
+	err = refreshWorkLoadPlanID(client, d)
+	if err != nil {
+		return []*schema.ResourceData{d}, fmt.Errorf("%s", err)
+	}
+
+	return []*schema.ResourceData{d}, nil
```
Comment (Lance52259) [inline huaweicloud/services/dws/resource_huaweicloud_dws_workload_plan.go]:
```suggestion
	return []*schema.ResourceData{d}, refreshWorkLoadPlanID(client, d)
```

Comment (jinyangyang222) [reply huaweicloud/services/dws/resource_huaweicloud_dws_workload_plan.go]: done.

**Pull request merged**